recipes/openslide/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.53.0"
```

Comment:
Will need >1.60 <2 || >=2.0.5 fopr this one as it's using the `<host_version>` syntax